### DISCUSSION TITLE
Avatar with automatic content

### DISCUSSION THREAD

**Discussion by philipp985** [mantinedev/mantine#6261] [Feature requests]
Hi,
thanks for this great library. Do you consider automatic generation of avatars as a part of the scope of this lib.

For example if I use the Avatar Component like this
```tsx
<Avatar name="John Doe" />
```

I would expect the avatar to have the initials "JD" and always the color lightBlue.

Here a some libraries that provide a concept for the expected result. Would you consider this as a good feature?
Examples:
- https://github.com/ambassify/react-avatar
- https://react.fluentui.dev/?path=/docs/components-avatar--default#name

**Comment by rtivital** [comment]
It is in the backlog with low priority

**Reply by philipp985** [reply]
Thanks for the reply.

**Comment by rtivital** [comment]
This feature will be available in 7.11.0 minor release.

**Reply by philipp985** [reply]
Awesome!
